### PR TITLE
RavenDB-14164 :  TimeSeries client API - include documents during timeseries loading

### PR DESCRIPTION
**Pull request by aviv86**
https://issues.hibernatingrhinos.com/issue/RavenDB-14164

### CODE REVIEW

File: src/Raven.Client/Documents/Session/AsyncSessionDocumentTimeSeries.cs [hunk 1]
```diff
@@ -31,10 +32,20 @@ public AsyncSessionDocumentTimeSeries(InMemoryDocumentSessionOperations session,
...
+        }
+
+        public Task<TimeSeriesEntry[]> GetAsync(DateTime? from, DateTime? to, Action<ITimeSeriesIncludeBuilder> includes, int start = 0, int pageSize = int.MaxValue, CancellationToken token = default)
+        {
+            return GetTimeSeriesAndIncludes<TimeSeriesEntry>(from, to, includes, start, pageSize, token);
         }
 
         public async Task<TTValues[]> GetAsyncInternal<TTValues>(DateTime? from = null, DateTime? to = null, int start = 0, int pageSize = int.MaxValue, CancellationToken token = default) where TTValues : TimeSeriesEntry
+        {
+            return await GetTimeSeriesAndIncludes<TTValues>(from, to, includes: null, start, pageSize, token).ConfigureAwait(false);
```
Comment (ppekrol) [inline src/Raven.Client/Documents/Session/AsyncSessionDocumentTimeSeries.cs]: no need to await, you can return the task, right?

File: src/Raven.Client/Documents/Session/Loaders/IncludeBuilder.cs [hunk 2]
```diff
@@ -341,6 +351,24 @@ IQueryIncludeBuilder<T> IQueryIncludeBuilder<T>.IncludeTimeSeries(Expression<Fun
             return this;
         }
 
+        public ITimeSeriesIncludeBuilder IncludeTimeSeries(string name, DateTime? from = null, DateTime? to = null)
```
Comment (ppekrol) [inline src/Raven.Client/Documents/Session/Loaders/IncludeBuilder.cs]: implement interface explicitly

File: src/Raven.Server/Documents/Handlers/TimeSeriesHandler.cs [hunk 3]
```diff
@@ -221,6 +227,10 @@ public async Task Read()
             if (pageSize == 0)
                 return rangeResultDictionary;
 
+            Dictionary<string, BlittableJsonReaderObject> includes = null;
+            if (includeDocument || includeTags)
```
Comment (ppekrol) [inline src/Raven.Server/Documents/Handlers/TimeSeriesHandler.cs]: can you wrap this inside a command like we have with other includes?

File: src/Raven.Client/Documents/Session/AsyncSessionDocumentTimeSeries.cs (hunk 1 above)
Comment (aviv86) [reply src/Raven.Client/Documents/Session/AsyncSessionDocumentTimeSeries.cs]: right, done

File: src/Raven.Client/Documents/Session/Loaders/IncludeBuilder.cs (hunk 2 above)
Comment (aviv86) [reply src/Raven.Client/Documents/Session/Loaders/IncludeBuilder.cs]: done

File: src/Raven.Server/Documents/Handlers/TimeSeriesHandler.cs (hunk 3 above)
Comment (aviv86) [reply src/Raven.Server/Documents/Handlers/TimeSeriesHandler.cs]: done

**Review by karmeli87 — APPROVED** [review]
LGTM

**Review by arekpalinski — APPROVED** [review]

**Pull request merged**